executor.py

```diff
@@ -26,6 +27,8 @@ def __init__(
 
         self._match_args = match_args or {}
         self._storage = DocumentArrayMemmap(self.workspace)
```

Comment:
```suggestion
        self._storage = DocumentArrayMemmap(self.workspace, key_length=kwargs.get('key_length', 64))
```

Reply:
Addressed, please see [this](https://github.com/jina-ai/executor-simpleindexer/pull/21/commits/a2a78c5be290e6d6d2f6f89b0ac117664084ed3f) commit for the change.